token/cli/tests/command.rs

```diff
@@ -3277,6 +3279,87 @@ async fn metadata_pointer(test_validator: &TestValidator, payer: &Keypair) {
     );
 }
 
+async fn group_pointer(test_validator: &TestValidator, payer: &Keypair) {
+    let program_id = spl_token_2022::id();
+    let config = test_config_with_default_signer(test_validator, payer, &program_id);
+    let group_address = Pubkey::new_unique();
+
+    let result = process_test_command(
+        &config,
+        payer,
+        &[
+            "spl-token",
+            CommandName::CreateToken.into(),
+            "--program-id",
+            &program_id.to_string(),
+            "--group-address",
+            &group_address.to_string(),
+        ],
+    )
+    .await;
+
+    let value: serde_json::Value = serde_json::from_str(&result.unwrap()).unwrap();
+    let mint = Pubkey::from_str(value["commandOutput"]["address"].as_str().unwrap()).unwrap();
+    let account = config.rpc_client.get_account(&mint).await.unwrap();
+    let mint_state = StateWithExtensionsOwned::<Mint>::unpack(account.data).unwrap();
+
+    let extension = mint_state.get_extension::<GroupPointer>().unwrap();
+
+    assert_eq!(
+        extension.group_address,
+        Some(group_address).try_into().unwrap()
+    );
+
+    let new_group_address = Pubkey::new_unique();
+
+    let _new_result = process_test_command(
+        &config,
+        payer,
+        &[
+            "spl-token",
+            CommandName::UpdateGroupAddress.into(),
+            &mint.to_string(),
+            &new_group_address.to_string(),
+        ],
+    )
+    .await;
```

Comment:
nit: It's better to unwrap these calls so that the test will fail loudly in the right place if there's an issue here